src/api/app/models/user.rb

```diff
@@ -675,6 +675,46 @@ def owned_packages
     owned
   end
 
+  def owned_item(owner, filters = {})
```

Comment:
could you please add specs to these new methods?

Reply:
Sure. On it.